### PR TITLE
Add disclaimer about matrix batch effects and normalization of CGMs (#18)

### PR DESCRIPTION
**Pull request by hannes-ucsc**

### CODE REVIEW

**Review by NoopDog — CHANGES_REQUESTED** [review]
Hi there, nice to see this moving forward. One comment, with the warning text in the project-level matrices paragraph, it seems to imply that the contributor-generated matrices are not normalized. Some may well be as far as I understand it.  We place no restrictions on contributor-generated matrices. Since they may be representing the result from a paper they may be highly normalized, filtered, tweaked, etc. to get the pretty picture. 

One suggestion to fix this is to, in the first paragraph, state there are three kinds of matrix files DCP/1-generated, DCP/2-generated, and contributor-generated, and that the DCP1 and DCP2-generated matrices are not normalized "within or across projects", and the contributor-generated matrices are whatever the contributor gave.

**Comment by clairerye** [comment]
100% agree with Dave's comment above, I don't think we should have a blanket statement. Can we use the metadata to distinguish if the statement should apply, or as Dave suggests a statement that covers the fact that CGMs are likely to be batch corrected etc?

**Review by aherbst-broad — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
@clairerye, from your agreement with @NoopDog it appears that you also intended to request changes. Could you please use the Github reject/approve functionality to make your choice explicit?

**Comment by hannes-ucsc** [comment]
@NoopDog 

> Hi there, nice to see this moving forward. One comment, with the warning text in the project-level matrices paragraph, it seems to imply that the contributor-generated matrices are not normalized. 

It states that the CGMs are 1) not corrected for batch effects and 2) not normalized *across* (!) projects.

I think it is safe to assume that 2 is the case because why would a contributor normalize against other HCA projects? If we get a pan-project matrix we need to invent a new category for those. Note the definition of *project-level* matrix at the beginning of the modified section.

Please explain why you think 1) does not apply.

> One suggestion to fix this is to, in the first paragraph, state there are three kinds of matrix files DCP/1-generated, DCP/2-generated, and contributor-generated, and that the DCP1 and DCP2-generated matrices are not normalized "within or across projects", and the contributor-generated matrices are whatever the contributor gave.

That would be scope creep. I will make clear in the issue that this PR only addresses a disclaimer about CGMs.

**Comment by clairerye** [comment]
If it is just about the CGMs then could we simply state that the DCP takes no responsibility for what, if any, batch correction or normalisation techniques have been used as they have been done directly by the contributor?

**Comment by hannes-ucsc** [comment]
That works for me. Do you want to suggest the specific wording using the suggest changes feature?

File: docs/dcp2_system_design.rst [hunk 1]
```diff
@@ -1004,7 +1004,9 @@ in those experiments.
 A matrix from a contributor or any other source external to the DCP is refered
 to as a *contributor-generated* matrix. This definition includes supporting
 files such as those containing cell type annotations, for example, that
-contributors may provide along with the actual matrices.
+contributors may provide along with the actual matrices. These matrices should
```
Comment (clairerye) [inline docs/dcp2_system_design.rst:1007]: DCP takes no responsibility for what, if any, batch correction or normalisation techniques have been used as they have been done directly by the data contributor.

File: docs/dcp2_system_design.rst [hunk 2]
```diff
@@ -1004,7 +1004,9 @@ in those experiments.
...
+be interpreted with caution: the data is not corrected for batch effects nor
```
Comment (clairerye) [inline docs/dcp2_system_design.rst]:
```suggestion
be interpreted with caution: DCP takes no responsibility for what, if any, batch correction or normalisation techniques have been used as they have been done directly by the data contributor.
```

**Review by clairerye — CHANGES_REQUESTED** [review]
suggested language changes

**Comment by clairerye** [comment]
oh no, did I actually comit the text change instead of just making the suggestion?

**Comment by hannes-ucsc** [comment]
No problem, I can revert the commit.

**Review by NoopDog — CHANGES_REQUESTED** [review]
Hmm, I thought the intent was to inform that DCP-generated matrices are not normalized within or across projects and that the DCP makes no representations about the normalization status of the contributor-generated matrices. This text only comments on the contributor-generated matrices and seems to leave out discussing the DCP-generated matrices.

**Comment by hannes-ucsc** [comment]
@clairerye I used your wording.

@NoopDog, maybe we can avoid blowing this up into something it wasn't intended to be?  Open a ticket and/or PR if you want more text to be added. Baby steps. Thank you!

**Review by NoopDog — APPROVED** [review]

**Review by ruchim — APPROVED** [review]
LGTM!

**Review by clairerye — APPROVED** [review]

**Comment by clairerye** [comment]
> Hmm, I thought the intent was to inform that DCP-generated matrices are not normalized within or across projects and that the DCP makes no representations about the normalization status of the contributor-generated matrices. This text only comments on the contributor-generated matrices and seems to leave out discussing the DCP-generated matrices.

I think given the title of the PR this was just about CGMs (or that is how I interpreted it). I am assuming we will have a different disclaimer for the DCP matrices.

**Review by kbergin — APPROVED** [review]

**Pull request merged**